src/java.desktop/share/classes/java/awt/MediaTracker.java

```diff
@@ -527,7 +526,7 @@ public boolean checkID(int id) {
      * @return      {@code true} if all images have finished loading,
      *                       have been aborted, or have encountered
      *                       an error; {@code false} otherwise
-     * @see         java.awt.MediaTracker#checkID(int, boolean)
+     * @see         java.awt.MediaTracker#checkID(int, boolean, boolean)
```

Comment:
int, boolean, boolean is a private method, it should probably refer the checkID(int)